File: apps/problems/isomorphic-strings/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    std::string s;
std::cin >> s;
std::string t;
std::cin >> t;
    Solution obj;
    bool result = obj.isIsomorphic(s, t);
    std::cout << (result ? "true" : "false") << std::endl;
    return 0;
}

File: apps/problems/isomorphic-strings/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;

    

    //##USER_CODE_HERE##

    public class Main {
      
    public static void main(String[] args) throws IOException {
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      String s = br.readLine().trim();
String t = br.readLine().trim();
      Solution obj = new Solution();
      boolean result = obj.isIsomorphic(s, t);
      System.out.println(result);
      br.close();
    }
      
    }
  